homepage/models.py

```diff
@@ -122,6 +134,17 @@ class FollowedUserCourses(models.Model):
     def __str__(self):
         return f'user = {self.user}, course = {self.course}'
 
+    @staticmethod
+    def get_courses_followed_by_appUser(appUser):

```

Comment:
python convention is:
https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables
please make sure to fix all variable names and method names according to the python convention
```suggestion
    def get_courses_followed_by_app_user(app_user):

```

Reply:
changed